src/pages/home/report/ReportActionsView.js

```diff
@@ -277,6 +271,22 @@ class ReportActionsView extends React.Component {
         Report.openReport(this.props.report.reportID);
     }
 
+    /**
+     * @returns {String|null}
+    */
+    computeNewMarkerReportActionID() {
+        let newMarkerReportActionID = null;
+        if (ReportUtils.isUnread(this.props.report)) {
+            const newMarkerIndex = _.findLastIndex(this.sortedReportActions, reportAction => (
+                (reportAction.reportActionTimestamp || 0) > this.props.report.lastReadTimestamp
+            ));
```

Comment:
I am a little confused about this logic and whether it works. `findLastIndex()` should iterate in reverse until we find an action that has a `timestamp` greater than the `lastReadTimestamp`. If there were 3 unread messages wouldn't it hit the last one first? But we want the 1 unread? Or does it work because `this.sortedReportActions` is backwards?

Reply:
Exactly. `this.sortedReportActions` is sorted in reverse order.